web/web.go

```diff
@@ -0,0 +1,42 @@
+package web
+
+import (
+	"net/http"
+	"io/ioutil"
+	"html/template"
+)
+
+func init() {
+	http.HandleFunc("/", rootHandler)
+	http.HandleFunc("/results", resultsHandler)
+}
+
+func rootHandler(writer http.ResponseWriter, request *http.Request) {
+	parseTemplate("search", writer, request)
+}
+
+func resultsHandler(writer http.ResponseWriter, request *http.Request) {
+	parseTemplate("results", writer, request)
+}
+
+func parseTemplate(templateName string, writer http.ResponseWriter, request *http.Request) {
+	htmlTemplate, err := loadTemplate(templateName, writer)
+	if err != nil {
+		http.Error(writer, err.Error(), http.StatusNotFound)
+		return
+	}
+	err = htmlTemplate.Execute(writer, "template")
```

Comment:
I thought it was the name of the template that was created with `template.New("template")`.